ONSdigital/census-rm-toolbox
# Add config for bulk invalid address processing

# Motivation and Context

We need to be able to process bulk invalid address events for case IDs from Address Resolution (AR)

# What has changed

* Added new processor for handling invalid addresses
* Yoinked validation for max length strings from sample-loader

# How to test?



1. Run Terraform in other PR to create bucket & permissions
1. Build image from this branch, push to your GCR
1. Deploy toolbox pod from other PR, ensuring you update the image to match your GCR image
1. Upload an invalid address CSV file to your bucket
1. Shell into the toolbox pod and run `bulkinvalidaddresses`
1. Check the case(s) address_invalid flag is set

# Links



https://trello.com/c/u9KwOLSz/1184-am-file-invalid-8
